graph/load.go

```diff
@@ -68,7 +68,7 @@ func (s *TagStore) CmdLoad(job *engine.Job) engine.Status {
 
 		for imageName, tagMap := range repositories {
 			for tag, address := range tagMap {
-				if err := s.Set(imageName, tag, address, true); err != nil {
+				if err := s.SetLoad(imageName, tag, address, true, job.Stdout); err != nil {
```

Comment:
@icecrime it depends ... if the tar ball being loaded has a new content i.e image ID's are different, but the image name: tag of the old and new images are same.

The idea is to show a display message to user, so that he doesn't get confused when the old image with the same name is renamed to none:none.